OpenEdXMobile/src/main/java/org/edx/mobile/view/VideoMoreOptionsBottomSheet.kt

```diff
@@ -0,0 +1,64 @@
+package org.edx.mobile.view
+
+import android.content.DialogInterface
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.viewModels
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.android.material.bottomsheet.BottomSheetDialog
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import dagger.hilt.android.AndroidEntryPoint
+import org.edx.mobile.databinding.BottomSheetVideoMoreOptionsBinding
+import org.edx.mobile.viewModel.VideoViewModel
+
+@AndroidEntryPoint
+class VideoMoreOptionsBottomSheet : BottomSheetDialogFragment() {
+
+    private lateinit var binding: BottomSheetVideoMoreOptionsBinding
+
+    private val videoViewModel: VideoViewModel by viewModels({ requireParentFragment() })
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = BottomSheetVideoMoreOptionsBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        (dialog as BottomSheetDialog).behavior.apply {
```

Comment:
as per [doc](https://m2.material.io/components/sheets-bottom#behavior) this behavior is not recommended. better to go with the default one.